contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1518,7 +1518,9 @@ union all
 -- details of user defined procedures
 select
     p.proname::sys.sysname as name 
-  , p.oid as object_id
+  , case
+      when t.typname = 'trigger' then tr.oid else p.oid
+    end as object_id
```

Comment:
do we need a similar change for the previous subquery as well ?

Reply:
The previous subquery is for system defined triggers/procedures, but there is no system defined trigger in TSQL. IMO we can skip that.